changelog/unreleased/features/2856--omit-empty-fields.md

```diff
@@ -0,0 +1,4 @@
+The JSON export format gained the options `--omit-empty-records`,
+`--omit-empty-lists`, and `--omit-empty-maps`, which cause empty records, lists,
+and maps not to be rendered respectively. The options may be combined together
+with the existing `--omit-nulls` option.
```

Comment:
I think we should have an `--omit-empty` short option that enables all 4. Nobody wants to type commands that long.

Reply:
I added this in the most recent commit.